eox_hooks/tests/test_models.py

```diff
@@ -0,0 +1,39 @@
+"""This file contains all the test for the general models.py file.
+
+Classes:
+    TestAuditModel: Test HookExecutionAudit.
+"""
+import time
+
+from django.test import TestCase
+
+from eox_hooks.constants import Status
+from eox_hooks.models import HookExecutionAudit
+
+
+class TestAuditModel(TestCase):
```

Comment:
since you are not changing the standard django behavior, I reckon these tests are not necessary, maybe test the __str__ method

Reply:
okok that's better

Reply:
I'm going to leave the first test so I can check that I can save task_duration using time.time()

Reply:
we already know that floatfield works with float values and time() returns a float ....